diff --git a/src/bdvmibackendfactory.cpp b/src/bdvmibackendfactory.cpp
index 0b08ab3..0976238 100644
--- a/src/bdvmibackendfactory.cpp
+++ b/src/bdvmibackendfactory.cpp
@@ -37,11 +37,11 @@ DomainWatcher *BackendFactory::domainWatcher()
 	}
 }
 
-Driver *BackendFactory::driver( const std::string &domain, bool watchableOnly )
+Driver *BackendFactory::driver( const std::string &domain, bool watchableOnly, bool useAltP2m )
 {
 	switch ( type_ ) {
 		case BACKEND_XEN:
-			return new XenDriver( domain, logHelper_, watchableOnly );
+			return new XenDriver( domain, logHelper_, watchableOnly, useAltP2m );
 		default:
 			throw std::runtime_error( "Xen is the only supported backend for now" );
 	}
